docs/data/routes/docs/fundamentals/services/dictionary/overview/en.md

```diff
@@ -0,0 +1,26 @@
+---
+name: overview
+routeTemplate: ./data/component-templates/article.yml
+title: Dictionary Services and APIs
+---
+
+# Dictionary Services and APIs
+
+Often multilingual apps will need a dictionary of static phrases that require translation. Commonly this would be items such as form labels, global navigation items, footers, etc. 
+
+## Sitecore endpoints for dictionary data
+
+You have several options for fetching dictionaries from Sitecore: 
+
+- [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service),
+- [Sitecore Experience Edge for XM](#sitecore-experience-edge-for-xm) or 
+- [Sitecore GraphQL](#sitecore-graphql).
+
+### Dictionary Service
+The [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service) is a REST endpoint providing app-specific dictionary data.
+### Sitecore Experience Edge for XM
+[Sitecore Experience Edge for XM](https://doc.sitecore.com/developers/101/developer-tools/en/introducing-sitecore-experience-edge-for-xm.html) comes with a read-only GraphQL schema that is designed to accommodate common front-end use cases for headless Sitecore development. Read about the [Experience Edge schema](https://doc.sitecore.com/developers/101/developer-tools/en/the-experience-edge-schema.html).
+### Sitecore GraphQL
```

Comment:
I would re-order these 3 so that #1 and #3 are together, like they are in my previous comment, since those 2 are both from Headless Services